Sentinel/Classes/Core/SourceScreenProvider.swift

```diff
@@ -13,6 +13,12 @@ public protocol SourceScreenProvider {
 
     /// The view controller used for presenting the Sentinel.
     func showTools(for view: some View)
+
+    /// Checks if Sentinel is already shown
+    func shouldShowSentinel() -> Bool
```

Comment:
In this case, based on the description of the method, I would rather go with something like `isSentinelShown`. Also, due to the usage of this, can we remove `Sentinel` from the name?